ament_cmake_core/cmake/core/package_xml_2_cmake.py

```diff
@@ -73,6 +76,20 @@ def get_dependency_values(key, depends):
     return dependencies
 
 
+def escape_back_slash(line):
```

Comment:
Actually I made this function when I thought the escaping was going to be more complicated, but I ended up with something really simple.

Maybe I should just do the replace in the `generate_cmake_code` function and drop the function all together. I also have two instances of the function because I didn't want to create a Python module common to both scripts.